docs/getting-started/plugins.md

```diff
@@ -0,0 +1,148 @@
+# Configure your plugins
+Screwdriver already defaults most configuration, but you can override defaults using a `local.yaml` or environment variables.
+
+## Datastore
+To use DynamoDB, use `dynamodb` plugin. To use Postgres, MySQL, and Sqlite, use `sequelize` plugin.
+
+#### DynamoDB
+Modify your `local.yaml`:
+```yaml
+datastore:
+    plugin: dynamodb
+    dynamodb:
+        # Prefix to the table names
+        prefix: TABLE-PREFIX
+        accessKeyId: AWS-ACCESS-KEY-ID
+        secretAccessKey: AWS-SECRET-ACCESS-KEY
+```
+
+Or set these environment variables in your `docker-compose.yml`:
+```yaml
+DATASTORE_PLUGIN: dynamodb
+DATASTORE_DYNAMODB_PREFIX: TABLE-PREFIX
+DATASTORE_DYNAMODB_ID: AWS-ACCESS-KEY-ID
+DATASTORE_DYNAMODB_SECRET: AWS-SECRET-ACCESS-KEY
+```
+
+#### Sequelize
+Modify your `local.yaml`:
+```yaml
+datastore:
+    plugin: sequelize
+    sequelize:
+        dialect: TYPE-OF-SERVER           # Ex: "sqlite"
+        storage: STORAGE-LOCATION         # Ex: "./database.sqlite"
+        database: DATABASE-NAME
+        username: DATABASE-USERNAME
+        password: DATABASE-PASSWORD
+        host: NETWORK-HOST
+        port: NETWORK-PORT
+```
+
+Or set these environment variables in your `docker-compose.yml`:
+```yaml
+DATASTORE_PLUGIN: sequelize
+DATASTORE_SEQUELIZE_DIALECT: TYPE-OF-SERVER
+DATASTORE_SEQUELIZE_DATABASE: DATABASE-NAME
+DATASTORE_SEQUELIZE_USERNAME: DATABASE-USERNAME
+DATASTORE_SEQUELIZE_PASSWORD: DATABASE-PASSWORD
+DATASTORE_SEQUELIZE_STORAGE: STORAGE-LOCATION
+DATASTORE_SEQUELIZE_HOST: NETWORK-HOST
+DATASTORE_SEQUELIZE_PORT: NETWORK-PORT
+```
+
+## Executor
+#### Kubernetes
+Modify your `local.yaml`:
+```yaml
+executor:
+    plugin: k8s
+    k8s:
+        # The host or IP of the kubernetes cluster
+        host: YOUR-KUBERNETES-HOST
+        token: JWT-FOR-AUTHENTICATING-KUBERNETES-REQUEST
+        launchVersion: LAUNCHER-VERSION
+```
+
+Or set these environment variables in your `docker-compose.yml`:
+```yaml
+EXECUTOR_PLUGIN: k8s
+K8S_HOST: YOUR-KUBERNETES-HOST
+K8S_TOKEN: JWT-FOR-AUTHENTICATING-KUBERNETES-REQUEST
+LAUNCH_VERSION: LAUNCHER-VERSION
+```
+
+#### Docker
+Modify your `local.yaml`:
+```yaml
+executor:
+    plugin: docker
+    docker:
+        docker:
+            __name: EXECUTOR_DOCKER_DOCKER
+            __format: json
+        launchVersion: LAUNCHER-VERSION
+```
+
+Or set these environment variables in your `docker-compose.yml`:
+```yaml
+EXECUTOR_PLUGIN: docker
+EXECUTOR_DOCKER_DOCKER: DOCKER-NAME
+LAUNCH_VERSION: LAUNCHER-VERSION
+```
+
+## Source Control
+
+### Step 1: Set up your OAuth Application
+You will need to set up an OAuth Application and retrieve your OAuth Client ID and Secret.
+
+#### Github:
+1. Navigate to the [Github OAuth applications](https://github.com/settings/developers) page.
+2. Click on the application you created to get your OAuth Client ID and Secret.
+3. Fill out the `Homepage URL` and `Authorization callback URL` with your `LoadBalancer Ingress`.
+
```

Comment:
How about running it locally? You don't really need `LoadBalancer Ingress`, right?